snap/snapcraft.yaml

```diff
@@ -120,12 +123,13 @@ parts:
             exit 1
         fi
         cd ..
-        mkdir -p $CRAFT_PART_INSTALL/bin
-        cp -r bin/* $CRAFT_PART_INSTALL/bin/
-        mkdir -p $CRAFT_PART_INSTALL/jars
-        cp -r jars/* $CRAFT_PART_INSTALL/jars/
-        mkdir -p $CRAFT_PART_INSTALL/python
-        cp -r python/* $CRAFT_PART_INSTALL/python/
+        BINS_DIR=$CRAFT_PART_INSTALL/opt/$CRAFT_PROJECT_NAME
```

Comment:
**todo** I believe it would be more appropriate `/opt/spark` for the spark binaries, and `/opt/k8s/` for `kubectl`. 

I mean, we should divide the two components (spark and kubectl), and spark folder should be names `spark`, as much as for charmed-kafka the folder `/opt/kafka` and not `/opt/charmed-kafka`